easybuild/easyconfigs/i/InterOp/InterOp-1.2.4-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'CMakeMake'
+
+name = 'InterOp'
+version = '1.2.4'
+
+homepage = 'https://illumina.github.io/interop/'
+description = """The Illumina InterOp libraries are a set of common routines used for
+reading InterOp metric files produced by Illumina sequencers including NextSeq
+1k/2k and NovaSeqX. These libraries are backwards compatible and capable of
+supporting prior releases of the software, with one exception: GA systems have
+been excluded."""
+
+software_license = 'LicenseGPLv3'
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```

Comment:
@SIB-software you need to move this up to `foss/2021a`, rather than specifying `SciPy` as you have done.
Otherwise this will break HMNS.

Reply:
Done